vnext/Shared/Modules/BlobModule.cpp

```diff
@@ -0,0 +1,538 @@
+    {".avif", "image/avif"},
+    {".avi", "video/x-msvideo"},
+    {".azw", "application/vnd.amazon.ebook"},
+    {".bin", "application/octet-stream"},
+    {".bmp", "image/bmp"},
+    {".bz", "application/x-bzip"},
+    {".bz2", "application/x-bzip2"},
+    {".cda", "application/x-cdf"},
+    {".csh", "application/x-csh"},
+    {".css", "text/css"},
+    {".csv", "text/csv"},
+    {".doc", "application/msword"},
+    {".docx", "application/vnd.openxmlformats-officedocument.wordprocessingml.document"},
+    {".eot", "application/vnd.ms-fontobject"},
+    {".epub", "application/epub+zip"},
+    {".gz", "application/gzip"},
+    {".gif", "image/gif"},
+    {".htm .html", "text/html"},
+    {".ico", "image/vnd.microsoft.icon"},
+    {".ics", "text/calendar"},
+    {".jar", "application/java-archive"},
+    {".jpeg .jpg", "image/jpeg"},
+    {".js", "text/javascript"},
+    {".json", "application/json"},
+    {".jsonld", "application/ld+json"},
+    {".mid .midi", "audio/midi audio/x-midi"},
+    {".mjs", "text/javascript"},
+    {".mp3", "audio/mpeg"},
+    {".mp4", "video/mp4"},
+    {".mpeg", "video/mpeg"},
+    {".mpkg", "application/vnd.apple.installer+xml"},
+    {".odp", "application/vnd.oasis.opendocument.presentation"},
+    {".ods", "application/vnd.oasis.opendocument.spreadsheet"},
+    {".odt", "application/vnd.oasis.opendocument.text"},
+    {".oga", "audio/ogg"},
+    {".ogv", "video/ogg"},
+    {".ogx", "application/ogg"},
+    {".opus", "audio/opus"},
+    {".otf", "font/otf"},
+    {".png", "image/png"},
+    {".pdf", "application/pdf"},
+    {".php", "application/x-httpd-php"},
+    {".ppt", "application/vnd.ms-powerpoint"},
+    {".pptx", "application/vnd.openxmlformats-officedocument.presentationml.presentation"},
+    {".rar", "application/vnd.rar"},
+    {".rtf", "application/rtf"},
+    {".sh", "application/x-sh"},
+    {".svg", "image/svg+xml"},
+    {".swf", "application/x-shockwave-flash"},
+    {".tar", "application/x-tar"},
+    {".tif .tiff", "image/tiff"},
+    {".ts", "video/mp2t"},
+    {".ttf", "font/ttf"},
+    {".txt", "text/plain"},
+    {".vsd", "application/vnd.visio"},
+    {".wav", "audio/wav"},
+    {".weba", "audio/webm"},
+    {".webm", "video/webm"},
+    {".webp", "image/webp"},
+    {".woff", "font/woff"},
+    {".woff2", "font/woff2"},
+    {".xhtml", "application/xhtml+xml"},
+    {".xls", "application/vnd.ms-excel"},
+    {".xlsx", "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet"},
+    {".xml", "application/xml"},
+    {".xul", "application/vnd.mozilla.xul+xml"},
+    {".zip", "application/zip"},
+    {".3gp", "video/3gpp; audio/3gpp if it doesn't contain video"},
+    {".3g2", "video/3gpp2; audio/3gpp2 if it doesn't contain video"},
+    {".7z", "application/x-7z-compressed"}}; // extensionToMime
+} // namespace
+
+namespace Microsoft::React {
+
+#pragma region BlobModule
+
+BlobModule::BlobModule(winrt::Windows::Foundation::IInspectable const &inspectableProperties) noexcept
```

Comment:
>BlobModule [](http://example.com/codeflow?start=12&length=10)

Julio, I don't understand how can someone use the Http/Fetch with blobs without implementing FileReaderModule !! Or am i missing something here ?

Reply:
[https://github.com/facebook/react-native/blob/1b31d6bb582768ccbe92d3c1a9e43354a8c531e5/Libraries/Blob/FileReader.js#L122](https://github.com/facebook/react-native/blob/1b31d6bb582768ccbe92d3c1a9e43354a8c531e5/Libraries/Blob/FileReader.js#L122)

Reply:
Will discuss offline.
This is likely a misunderstanding of the Android code.

Reply:
FileReader module has been implemented and tested (See `BobTest.js`).